share/dbtypes_gen.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import os
+import sys
+import json
+
+with open(os.path.join(os.path.dirname(__file__), '..', 'src', 'query_backend.h')) as f:
+    lines = f.readlines()
+
+VERSION = "v"+sys.argv[1]
+
+ALL_BACK = ["SQLite", "HDF5"]
+
+ENUM_START = 0
+
+for i in range(0,len(lines)):
+    if "enum DbTypes {" in lines[i]:
+        ENUM_START = i
```

Comment:
Add a `break` after this line.  No need to iterate through the rest of the lines once we have the start loc.